microprofile/server/src/main/java/io/helidon/microprofile/server/Server.java

```diff
@@ -69,6 +71,20 @@ static Server create(Class<? extends Application>... applicationClasses) throws
         return builder.build();
     }
 
+    /**
+     * Create a server instance using a Websocket application class.
+     *
+     * @param applicationClass websocket application class
+     * @return server instance to be started
+     * @throws MpException in case the server fails to be created
+     * @see #builder()
+     */
+    static Server create(Class<? extends ServerApplicationConfig> applicationClass) throws MpException {
```

Comment:
WebSocket support should not be part of Helidon MicroProfile server.
This adds an unwanted dependency on Tyrus for MP implementation.
Please extract WebSocket support into a separate module.